notebook/tests/services/session.js

```diff
@@ -164,6 +167,9 @@ casper.notebook_test(function () {
         20000
     );
 
+    this.thenEvaluate( function() {IPython.notebook.session.start()});
+    this.wait_for_kernel_ready();
+
```

Comment:
These tests were assuming that a `kernel_killed.Session` event will be triggered, even in cases where the kernel was already killed (which was the case here, the kernel is dead at the beginning of these two event tests). I've cleaned up the code so that `kernel_killed.Session` event is only triggered when the kernel is actually being killed. So here we first restore the notebook session so the kernel is running, restoring the expected events.